prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -432,60 +433,11 @@
 
 </section>
 
-<section class="mb-4">
-
-  <app-page-subheader>
-    <ng-container appPageSubheaderTitle>Self-declaration</ng-container>
-
-    <button *ngIf="showEditRedirect"
-            mat-icon-button
-            matTooltip="Edit Self-declaration"
-            (click)="onRoute(EnrolmentRoutes.SELF_DECLARATION)">
-      <mat-icon>edit</mat-icon>
-    </button>
-  </app-page-subheader>
-  <app-enrollee-property [title]="selfDeclarationQuestions[SelfDeclarationTypeEnum.HAS_REGISTRATION_SUSPENDED]"
-                         [makeBold]="false">
-    {{ hasRegistrationSuspended() | yesNo: true }}
-  </app-enrollee-property>
-
-  <app-enrollee-property *ngIf="hasRegistrationSuspended()"
-                         title="Description"
-                         [makeBold]="false">
-    {{ getRegistrationSuspendedDetails() }}
-  </app-enrollee-property>
-
-  <app-enrollee-property [title]="selfDeclarationQuestions[SelfDeclarationTypeEnum.HAS_CONVICTION]"
-                         [makeBold]="false">
-    {{ hasConviction() | yesNo: true }}
-  </app-enrollee-property>
-
-  <app-enrollee-property *ngIf="hasConviction()"
-                         title="Description"
-                         [makeBold]="false">
-    {{ getConvictionDetails() }}
-  </app-enrollee-property>
-
-  <app-enrollee-property [title]="selfDeclarationQuestions[SelfDeclarationTypeEnum.HAS_PHARMANET_SUSPENDED]"
-                         [makeBold]="false">
-    {{ hasPharmaNetSuspended() | yesNo: true }}
-  </app-enrollee-property>
-
-  <app-enrollee-property *ngIf="hasPharmaNetSuspended()"
-                         title="Description"
-                         [makeBold]="false">
-    {{ getPharmaNetSuspendedDetails() }}
-  </app-enrollee-property>
-
-  <app-enrollee-property [title]="selfDeclarationQuestions[SelfDeclarationTypeEnum.HAS_DISCIPLINARY_ACTION]"
-                         [makeBold]="false">
-    {{ hasDisciplinaryAction() | yesNo: true }}
-  </app-enrollee-property>
-
-  <app-enrollee-property *ngIf="hasDisciplinaryAction()"
-                         title="Description"
-                         [makeBold]="false">
-    {{ getDisciplinaryActionDetails() }}
-  </app-enrollee-property>
-
-</section>
+<app-enrollee-self-declarations [enrolment]="enrolment">
```

Comment:
@neophyte57 @hogmann @AnaisHebert 
To help speed up reviewing the most recent changes are related to refactoring of enrollee self declarations and addition of showing/downloading of self declaration documents are encapsulated in a new component, and removed from this components controller and template.